clients/admin-ui/src/features/common/CheckboxTree.tsx

```diff
@@ -70,7 +70,7 @@ interface CheckboxItemProps {
   onExpanded: (node: TreeNode) => void;
   isIndeterminate: boolean;
   isDisabled: boolean;
-  children?: ReactNode;
+  children?: ReactNode[] | JSX.Element[] | null;
```

Comment:
Why was this necessary, out of curiosity?

Reply:
great question. It was the fact that the `children` being passed was a ternary that sometimes resulted in `null`. I just took a new look at it and realized that if we return `undefined` instead, we don't need this extra effort on the typing. Good call out!